spinalcordtoolbox/scripts/sct_get_centerline.py

```diff
@@ -214,11 +215,21 @@ def main(argv: Sequence[str]):
         return
 
     # Extrapolate and regularize (or detect if optic) cord centerline
-    im_centerline, arr_centerline, _, _ = get_centerline(im_labels,
-                                                         param=param_centerline,
-                                                         space=arguments.space,
-                                                         verbose=verbose,
-                                                         remove_temp_files=arguments.r)
+    im_centerline, _, _, _ = get_centerline(im_labels,
+                                            param=param_centerline,
+                                            # NB: We intentionally skip the pix->phys conversion here, since we want to
+                                            #     convert to phys AFTER the image has been reoriented away from RPI.
+                                            # space=arguments.space,
+                                            verbose=verbose,
+                                            remove_temp_files=arguments.r)
+
+    # Fetch `arr_centerline` from `im_centerline`
+    # (This is the easiest way to get coords that are in their original orientation. This is largely equivalent to how
+    #  arr_centerline is determined inside `get_centerline`, but without the RPI orientation.)
+    arr_centerline = np.array(find_and_sort_coord(im_centerline))
```

Comment:
This is true for the `optic` method (the default), or when `param.soft == 1` (_not_ the default), but in other cases will result in centerline coordinates being rounded to integer values. Is this loss of precision ok?

I guess the CSV is informative only, and this is a quick fix before looking at whether `get_centerline` has a deeper bug.